### ISSUE TITLE
Error when parsing floats with commas

### ISSUE THREAD

**Issue by girazoki** [witnet/witnet-rust#1133]
Source: [https://www.chards.co.uk/data/update-prices?full=false&currency=eur]

Script:
```js
const chards = new Witnet.Source("https://www.chards.co.uk/data/update-prices?full=false&currency=eur")
  .parseMapJSON()
  .getMap("gold")
  .getFloat("gram")
  .multiply(1000)
  .round()
```

Error:
> `RadError` cannot be converted to `RadonError` but it should, because it is needed for the tally result. Message: None. Inner: `Some(ParseFloat { message: "invalid float literal" })`

**Comment by tmpolaczyk** [comment]
That's not supported, a simple workaround would be to `String.replace(",", "")`

**Comment by aesedepece** [comment]
> That's not supported, a simple workaround would be to `String.replace(",", "")`

If we only had a `.replace` RADON method...

**Comment by drcpu-github** [comment]
@aesedepece Would it make sense to have a getFloat RADON function that allows to specify thousand and decimal separators for greater parsing flexibility? Example `getFloat("gram", thousand=" ", decimal=",")` for a number `111 111,111`.

**Comment by Tommytrg** [comment]
Could we create a WIP for this? @tmpolaczyk @lrubiorod

**Comment by tmpolaczyk** [comment]
We could

**Comment by aesedepece** [comment]
Yes, we could

**Comment by aesedepece** [comment]
This was solved by #2235, now about to be released inside 1.6 as per [WIP-0024](https://github.com/witnet/WIPs/blob/master/wip-0024.md).
